src/chunktransformer.ts

```diff
@@ -147,17 +153,6 @@ export class ChunkTransformer extends Transform {
         default:
           break;
       }
-      if (this._destroyed) {
-        return;
-      }
-    }
-    if (data.lastScannedRowKey && data.lastScannedRowKey.length > 0) {
-      this.lastRowKey = Mutation.convertFromBytes(
-        data.lastScannedRowKey as Bytes,
-        {
-          userOptions: this.options,
-        }
-      );
     }
```

Comment:
I think this breaks request resumption logic in makeNewRequest

`const lastRowKey = chunkTransformer ? chunkTransformer.lastRowKey : '';`

Reply:
Fixed this